### PR TITLE
CSUB-962, CSUB-963, CSUB-959, CSUB-969, CSUB-974: Polkadot Release v1.1.0 patch

### PR DESCRIPTION
**Pull request by frankli-dev**
# Description of proposed changes

<describe what this PR is about and why we want it>

---

Practical tips for PR review & merge:

- [ ] All GitHub Actions report PASS
- [ ] Newly added code/functions have unit tests
  - [ ] Coverage tools report all newly added lines as covered
  - [ ] The positive scenario is exercised
  - [ ] Negative scenarios are exercised, e.g. assert on all possible errors
  - [ ] Assert on events triggered if applicable
  - [ ] Assert on changes made to storage if applicable
- [ ] Modified behavior/functions - try to make sure above test items are covered
- [ ] Integration tests are added if applicable/needed

### CODE REVIEW

**Review by nathanwhit — APPROVED** [review]

**Review by atodorov — CHANGES_REQUESTED** [review]
Need to update the whitelist for allowed forks to allow gluwa/frontier and gluwa/polkadot-sdk repositories.  This will fix the failure reported in https://github.com/gluwa/creditcoin3/actions/runs/7254387992/job/19763009568?pr=132. 

The rest looks good so far. Retesting Staking Dashboard against this PR!

File: Cargo.toml [hunk 1]
```diff
@@ -51,126 +51,126 @@ thiserror = "1.0"
...
-sc-rpc = { version = "4.0.0-dev", git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.1.0" }
-sc-rpc-api = { version = "0.10.0-dev", git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.1.0" }
-sc-service = { version = "0.10.0-dev", git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.1.0", default-features = false }
-sc-telemetry = { version = "4.0.0-dev", git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.1.0" }
-sc-transaction-pool = { version = "4.0.0-dev", git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.1.0" }
-sc-transaction-pool-api = { version = "4.0.0-dev", git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.1.0" }
-sc-utils = { version = "4.0.0-dev", git = "https://github.com/paritytech/polkadot-sdk", branch = "release-polkadot-v1.1.0" }
+sc-basic-authorship = { version = "0.10.0-dev", git = "https://github.com/gluwa/polkadot-sdk", branch = "release-polkadot-v1.1.0-patch" }
```
Comment (atodorov) [inline Cargo.toml:54]: Opening a comment to track progress of staking dashboard retest against this PR:

**Review by atodorov — CHANGES_REQUESTED** [review]

File: Cargo.toml (hunk 1 above)
Comment (atodorov) [reply Cargo.toml:54]: Unbonding from a pool now seems to show the correct amounts everywhere:

- when initiating the unbond
01_unbond_shows_correct_amount

- on the progress bar
02_unlocking_progress_bar_shows_correct_amount

- in the unlock modal
03_unlock_modal_shows_correct_amounts

- in Polakdot JS Apps
04_polkadot_apps_shows_correct_amount

- unlocked funds shows correct amount
05_unlocked_funds_shows_correct_amount

- withdrawing also shows correct amount
06_witdraw_modal_shows_correct_amount

- total funds after withdraw looks correct as well (there were 33 CTC rewards claimed + fees)
07_total_balance_after_withdraw_shows_correct

**Comment by atodorov** [comment]
See https://github.com/gluwa/creditcoin3/pull/132#discussion_r1431291054 for detailed test results + screenshots!

**Review by atodorov — APPROVED** [review]

**Pull request merged**